scripts/requireDeps.sh

```diff
@@ -4,7 +4,7 @@ set -ue
 # Check if an application is on the PATH.
 # If it is not, return with non-zero.
 _is_dep_available() {
-	command -v "$1" >/dev/null || (echo "$1 ${2:-was not found and is a required dependency}"; return 1)
+	command -v "$1" >/dev/null || (echo "\"$1\" ${2:-package was not found and is a required dependency}"; return 1)
```

Comment:
`package` implies that this program is in that exact package on every system, which it may not be. Perhaps `tool` would be better?

Reply:
Quite frankly, `command` would be most ideal.